lib/hostel/site.rb

```diff
@@ -22,7 +22,7 @@ def build_path(path, secure: true, path_args: {})
         'http://'
       end
 
-      url << domain
+      url << ENV['QA_HOST']?ENV['QA_HOST']:domain
```

Comment:
Is your space bar broken?
If not, and you are just checking if `ENV['QA_HOST']` is nil, might be easier with:
`ENV['QA_HOST'] || domain`